### DISCUSSION TITLE
Not all results are displayed

### DISCUSSION THREAD

**Discussion by karenpayneoregon** [community/community#9949] [Other Feature Feedback, Questions, & Ideas]
Entered `repo:OED-Applications/mobile-friendly-claims redirectLocation` which came up with correct results, showed only two (out of 48) and indicated the indexing is not fully there yet.

Will say nice execution time :thumbsup:

**Comment by colinwm** [comment]
That was 24 hours ago, can you try again? Hopefully we have finished indexing that repository by now.
